CMakeLists.txt

```diff
@@ -20,29 +23,81 @@ add_compile_definitions(TFLITE_BUILD_WITH_XNNPACK_DELEGATE)
 
 # This assumes that tensorflow got checked out as submodule.
 # True, if `git clone` had the additional option `--recursive`.
-add_subdirectory(tensorflow/tensorflow/lite 
+add_subdirectory(tensorflow/tensorflow/lite
   "${CMAKE_CURRENT_BINARY_DIR}/tensorflow-lite" EXCLUDE_FROM_ALL)
 
 add_compile_definitions(DEEPSEG_VERSION=${DEEPSEG_VERSION})
 
+add_library(backscrub
+  libbackscrub.cc
+  transpose_conv_bias.cc)
```

Comment:
Maybe split out the lib sources into a `lib/` or `src/lib/` sub directory? Thus moving all source files out from the main source directory.

Reply:
Sure - `lib/` and `app/` ok?